workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -249,6 +249,46 @@
     end
   end
 
+  describe '#list' do
+    build_type_json_str = '{"buildType":[{"id":"Wildcard_A","projectName":"Wildcard :: A"},
+                                         {"id":"B_Wildcard","projectName":"Wildcard :: B"}]}'
+    before do
+      stub_request(:get, /buildTypes/).to_return(:status => 200, :body => build_type_json_str)
+    end
+
+    it 'replies the correct builds if wildcard exists' do
+      wildcard = "Wildcard_A"
```

Comment:
Since we using `include? ` with  `wildcard` we should check `wildcard = "Wildcard"` instead. also add a `buildType.id` that doesn't contain wildcard and verify it's not in the reply.